src/biogeochem/DryDepVelocity.F90

```diff
@@ -102,13 +102,15 @@ Module DryDepVelocity
   !------------------------------------------------------------------------
   subroutine Init(this, bounds)
 
-    use clm_varctl     , only : use_fates, use_fates_sp
+    use clm_varctl     , only : use_fates, use_fates_sp, use_fates_nocomp
     class(drydepvel_type) :: this
     type(bounds_type), intent(in) :: bounds
 
-    if ( (.not. use_fates_sp) .and. use_fates .and. (n_drydep > 0) ) then
-       call endrun( msg='ERROR: Dry-deposition currently does NOT work with FATES outside of FATES-SP mode (see github issue #1044)'//&
+    if (use_fates .and. (n_drydep > 0)) then    
```

Comment:
when SP is on NOCOMP is also on by definition, so this can be simplified...

Reply:
So I'm confused here. Don't we want to check for use_fates_nocomp - since just use_fates could be problematic with dry deposition?

Reply:
Please ignore my comment - I was looking at the wrong line.